src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/utils.ts

```diff
@@ -0,0 +1,172 @@
+import produce from 'immer';
+import { Updater } from 'use-immer';
+
+import { NAME_INPUT_FIELD } from '@catalog/templatescatalog/utils/consts';
+import { TabsData } from '@catalog/utils/WizardVMContext/utils/tabs-data';
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import DataVolumeModel from '@kubevirt-ui/kubevirt-api/console/models/DataVolumeModel';
+import { V1beta1DataVolume } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { V1VirtualMachine, V1Volume } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  DEFAULT_CDROM_DISK_SIZE,
+  DEFAULT_DISK_SIZE,
+} from '@kubevirt-utils/components/DiskModal/state/initialState';
+import { DEFAULT_NAMESPACE } from '@kubevirt-utils/constants/constants';
+import { UploadDataProps } from '@kubevirt-utils/hooks/useCDIUpload/useCDIUpload';
+import { getTemplateParameterValue } from '@kubevirt-utils/resources/template';
+import { getVolumes } from '@kubevirt-utils/resources/vm';
+import { ensurePath, isEmpty } from '@kubevirt-utils/utils/utils';
+
+import { INSTALLATION_CDROM_NAME } from './StorageSection/constants';
+
+export const allRequiredParametersAreFullfilled = (template: V1Template) =>
+  template.parameters.every((parameter) => (parameter.required ? parameter.value !== '' : true));
+
+export const doNameParameterExists = (template: V1Template): boolean =>
+  !isEmpty(template?.parameters) &&
+  !isEmpty(template?.parameters?.find((param) => param?.name === NAME_INPUT_FIELD));
```

Comment:
no need for !isEmpty(template?.parameters) ,
simply do 
!isEmpty((template?.parameters || [])?.find((param) => param?.name === NAME_INPUT_FIELD));